pymatgen/io/pwmat/inputs.py

```diff
@@ -47,18 +47,19 @@ class ListLocator(MSONable):
     """Find the element indices (starts from 0) of a certain paragraph of text from the list."""
 
     @staticmethod
-    def locate_all_lines(strs_lst: list[str], content: str) -> list[int]:
+    def locate_all_lines(strs_lst: list[str], content: str, exclusion: str | None = None) -> list[int]:
```

Comment:
thanks @lhycms! 👍 we need a test for the new `exclusion` keyword

Reply:
Hi @janosh, I have uploaded the latest version of `MOVEMENT` file, and add tests for `exclusion` keyword.